pkg/log/zap/flags.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zap
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/pflag"
+
+	"go.uber.org/zap/zapcore"
+
```

Comment:
Nit: What is the reason for all the new lines in between the imports?

Reply:
None at all. I'll fix this.